### ISSUE TITLE
Failed to execute 'selectNodeContents' on 'Range': parameter 1 is not of type 'Node

### ISSUE THREAD

**Issue by LorenzKeel** [editor-js/attaches#42]
Hi!

I have an error after uploading a file: "failed to execute 'selectNodeContents' on 'Range': parameter 1 is not of type 'Node'".

The error occurs in version 1.1.1 and I don't have it in version 1.1.0

**Comment by jerinjacob007** [comment]
Same happens with me.
Entire editorjs becomes undefined.

I'm using react editorjs.
Previously there was no errors.

**Comment by jerinjacob007** [comment]
**Solution**
Add a title field to file object in response 
`file: {
//other response objects,
title : filename
}`

This happens probably because of the changes in [84fb3](https://github.com/editor-js/attaches/commit/84fb3ced1679ecabb2c0c8c3315b1a08d6003240)

The **title** field was response file name by default and I think it's changed in the above mentioned PR

**Comment by ChukwuebukaN** [comment]
> **Solution** Add a title field to file object in response `file: { //other response objects, title : filename }`
> 
> This happens probably because of the changes in [84fb3](https://github.com/editor-js/attaches/commit/84fb3ced1679ecabb2c0c8c3315b1a08d6003240)
> 
> The **title** field was response file name by default and I think it's changed in the above mentioned PR

Seems the problem currently isn't that of file name being present, as my implementation uploads the file and previews it.. then show's the error afterwards.
```
   return {
      success: 1,
      file: {
        url: isFile.link,
        size: isFile.size,
        name: isFile.name,
        extension: isFile.type,
      },
      title: "Hero",
    };
```
I can see and click on the attachment, and it downloads, but the error pops up anyways.

**Comment by jerinjacob007** [comment]
> > **Solution** Add a title field to file object in response `file: { //other response objects, title : filename }`
> > This happens probably because of the changes in [84fb3](https://github.com/editor-js/attaches/commit/84fb3ced1679ecabb2c0c8c3315b1a08d6003240)
> > The **title** field was response file name by default and I think it's changed in the above mentioned PR
> 
> Seems the problem currently isn't that of file name being present, as my implementation uploads the file and previews it.. then show's the error afterwards.
> 
> ```
>    return {
>       success: 1,
>       file: {
>         url: isFile.link,
>         size: isFile.size,
>         name: isFile.name,
>         extension: isFile.type,
>       },
>       title: "Hero",
>     };
> ```
> 
> I can see and click on the attachment, and it downloads, but the error pops up anyways.

The title field is still outside `file` object can you try like this 
```
file:{
...url,size,name,
title: "Hero"
}
```

**Comment by ChukwuebukaN** [comment]
> > > **Solution** Add a title field to file object in response `file: { //other response objects, title : filename }`
> > > This happens probably because of the changes in [84fb3](https://github.com/editor-js/attaches/commit/84fb3ced1679ecabb2c0c8c3315b1a08d6003240)
> > > The **title** field was response file name by default and I think it's changed in the above mentioned PR
> > 
> > 
> > Seems the problem currently isn't that of file name being present, as my implementation uploads the file and previews it.. then show's the error afterwards.
> > ```
> >    return {
> >       success: 1,
> >       file: {
> >         url: isFile.link,
> >         size: isFile.size,
> >         name: isFile.name,
> >         extension: isFile.type,
> >       },
> >       title: "Hero",
> >     };
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > I can see and click on the attachment, and it downloads, but the error pops up anyways.
> 
> The title field is still outside `file` object can you try like this
> 
> ```
> file:{
> ...url,size,name,
> title: "Hero"
> }
> ```

@jerinjacob1999 God Bless You bro!, This works perfectly now, I guess **@editor-js** should update their "**_official_**" documentation.

**Comment by neSpecc** [comment]
Should be resolved by #57
